packages/central-server/src/hooks/entityHouseholdHead.js

```diff
@@ -0,0 +1,19 @@
+import { updateJsonFieldValue } from './utilities';
+
+export async function entityHouseholdHead({ answer, surveyResponse }) {
```

Comment:
This is a really tidy hook! There's the potential to take it further by allowing it to set any entity attribute, what do you think about having a go at that?

You'd require the question config to include a field like `entityAttribute`, and then in here you'd query the answer for its question and get the attribute to set from that.

Reply:
@edmofro do you mean that there would be a property associated with a question called 'entityAttribute' with a value which would be end up being the 'key' in the attribute field along with the string value of the answer? Where would that property be set?

Reply:
That's right! I was thinking the property would be set in `config`, but I've just checked and that's `survey_screen_component.config` not `question.config`. We could add a new `hook_config` column to `question`, or we could abuse the hook name itself using some convention like calling the hook `entityAttributeHouseholdHead`, where any hook starting with `entityAttribute` resolves to the same hook handler

Reply:
@edmofro I've added another couple of commits, with the ability to add custom entity attributes. Let me know what you think.